### DISCUSSION TITLE
Bitcoin Design Twitter Presence

### DISCUSSION THREAD

**Discussion by pavlenex** [BitcoinDesign/Meta#79] [Ideas]
It's been mentioned on a few calls already that we should probably begin communicating our work a bit better and make wider Bitcoin community aware of our work and existence.

We as a community have a placeholder Twitter account https://twitter.com/bitcoin_design

Some topics worth disusing:

- Who runs the twitter account
- Who can post from the account (maybe via tweetdeck or do we have an opened process via GitHub where anyone can submit a tweet?)
- What kind of content do we want to share (regular updates, education content?)
- What tone/personality should it have?

Generally, after discussing with few contributors, my impression is that we want a more neutral, education-orientated twitter presence, that can educate people about Bitcoin by communicating our work alongside. One example that good mentioned is that we can share educational content but in very visual way. Think of it as tweets https://twitter.com/dannydiekroeger posts but focused on visuals. 

Tweets can be a good way for designers to get involved and do something simple (design a visual for a tweet), get credited for it and help people learn about particular topic.

**Comment by moneyball** [comment]
I think we should have a friendly tone. Not an attack dog. Not shitting on other coins or other views. Just stay focused on bitcoin design and speak positively about bitcoin design.

The group of people with access to the Twitter account should definitely be on the same page for style and content plan in order to have a consistent voice. And of course that should reflect the desires of the community.

**Comment by GBKS** [comment]
Also check these previous documents with thoughts on the topic - [here](https://hackmd.io/WpZ768_LQMeYvKoGOYLgHQ) and [here](https://docs.google.com/document/d/1KVrx1-LoiXlvNXW3ocOgAn7Pm5hQIWGswDLmLt7Dk8I/edit?usp=sharing). The sentiment to post as a champion of bitcoin design and reflect community activity seems to be consistent.

As far as who posts, this could be an open question for the community. My suggestion would be to tie it in with the newsletter effort which is similar in many respects. Maybe we can just create a social-media channel on Slack. People can post things they think are Twitter-worthy, and we use thumbs up/down to vote, figure out the tweet content in the visuals, etc.

**Comment by pavlenex** [comment]
One more educational-orientated account I personally like is https://twitter.com/DocumentingBTC

**Comment by Bosch-0** [comment]
Keeping a neutral tone that only acts a feed of what the community is up to / design related content I think is the best. No opinion, political, satirical etc. based tweets.
